CIP-0001/README.md

```diff
@@ -1,185 +1,374 @@
 ---
 CIP: 1
-Title: CIP process
-Authors: Frederic Johnson <frederic.johnson@cardanofoundation.org>, Sebastien Guillemot <sebastien@dcspark.io>, Matthias Benkort <matthias.benkort@cardanofoundation.org>, Duncan Coutts <duncan.coutts@iohk.io>
+Title: Cardano Improvement Proposals
 Status: Active
-Type: Process
-Created: 2020-03-21
+Category: Meta
+Authors:
+    - Matthias Benkort <matthias.benkort@cardanofoundation.org>
+    - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Implementors: N/A
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/365
+Created: 2022-10-14
```

Comment:
While this expands on the original CIP, I think the date should be that of the initial proposal here. It might be confusing to some to see dates all over the place since the process is functionally the same